db/migrate/20201109080907_create_read_texts.rb

```diff
@@ -0,0 +1,10 @@
+class CreateReadTexts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :read_texts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
```

Comment:
ここが `post` が出てくる原因ではありませんか？

```suggestion
      t.references :movie, null: false, foreign_key: true
```

ただ，その前に動画に対して視聴済み機能をつけるならば，命名がread_textsはよろしくないでしょう。
モデル名・テーブル名から修正すべきです。

Reply:
ただ，その前に動画に対して視聴済み機能をつけるならば，命名がread_textsはよろしくないでしょう。
モデル名・テーブル名から修正すべきです。

なるほどです。では特にこだわりも無いので再度修正が必要になるのであれば、aws_textで進めます！２転することお許しください＞＜失礼致しました！！

再度aws_textへ内容を戻し `rails db:migrate`を行いました。ビューへの記述も `aws_texts/show.html.erb`へ変更し記述内容を修正したところ上手く☆マークが表示されました！！続きを進めてみます！ありがとうございます！！